style.css

```diff
@@ -12,20 +11,32 @@
 
 @font-face {
   font-family: 'Montserrat-Light';
-  src: url('fonts/Montserrat-Light.eot?#iefix') format('embedded-opentype'),  
+  src: url('fonts/Montserrat-Light.eot?#iefix') format('embedded-opentype'),
 	url('fonts/Montserrat-Light.otf')  format('opentype'),
-	url('fonts/Montserrat-Light.woff') format('woff'), 
-	url('fonts/Montserrat-Light.ttf')  format('truetype'), 
+	url('fonts/Montserrat-Light.woff') format('woff'),
+	url('fonts/Montserrat-Light.ttf')  format('truetype'),
 	url('fonts/Montserrat-Light.svg#Montserrat-Light') format('svg');
   font-weight: normal;
   font-style: normal;
 }
 
-body {background-color: red; font: 36px "Montserrat-Light";}
+body {
+  font: 36px "Montserrat-Light";
```

Comment:
Add backup font family for backwards compatibility. e.g.
`font-family: 'Montserrat-Light', sans-serif;`